### ISSUE TITLE
All 500 if TracingService uses service with FQDN

### ISSUE THREAD

**Issue by marcantoine-bibeau** [datawire/ambassador#3021]
**Describe the bug**
Ambassador return 500 on all requests when TracingService is configured with something like 
```service: datadog.default.svc.cluster.local:8126``` but works fine if configured with ```service: datadog:8126```

**To Reproduce**
1. Go to 'http://ambassador-admin.ambassador.svc.cluster.local:8877/ambassador/v0/diag/'
2. Receiving 500
4. See error in logs
```
2020-10-15 15:31:59 diagd 1.7.4 [P74TThreadPoolExecutor-0_4] ERROR: 'dict object' has no attribute 'cluster_tracing_datadog_default_svc_cluster_local_8126_ambassador'
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ambassador-0.0.0.dev0-py3.7.egg/ambassador_diag/diagd.py", line 554, in wrapper
    result = f(*args, reqid=reqid, **kwds)
  File "/usr/lib/python3.7/site-packages/ambassador-0.0.0.dev0-py3.7.egg/ambassador_diag/diagd.py", line 936, in show_overview
    return Response(render_template("overview.html", **tvars))
  File "/usr/lib/python3.7/site-packages/flask/templating.py", line 140, in render_template
    ctx.app,
  File "/usr/lib/python3.7/site-packages/flask/templating.py", line 120, in _render
    rv = template.render(context)
  File "/usr/lib/python3.7/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/usr/lib/python3.7/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/usr/lib/python3.7/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/usr/lib/python3.7/site-packages/ambassador-0.0.0.dev0-py3.7.egg/templates/overview.html", line 150, in top-level template code
    <code><span style="color:{{ cluster_stats[service.cluster].hcolor }}">
  File "/usr/lib/python3.7/site-packages/jinja2/environment.py", line 471, in getattr
    return getattr(obj, attribute)
```

**Expected behavior**
should access diag page

**Versions (please complete the following information):**
 - Ambassador: 1.7.4, 1.8.0
 - Kubernetes environment [EKS]

**Additional context**
none

**Comment by esmet** [comment]
Hey @marcantoine-bibeau - thanks for reporting this.

I believe I this is related to the same underlying issue as https://github.com/datawire/ambassador/issues/3019 and I'm working on a fix. For now, please continue to use the shorter service name to work around the issue.

**Comment by khussey** [comment]
This is fixed in the 1.8.1 release, which is now available.
